widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/alert/AlertRemoteConfig.kt

```diff
@@ -32,6 +32,9 @@ internal data class AlertRemoteConfig(
     @SerializedName("negativeButton")
     val negativeButtonRemoteConfig: ButtonRemoteConfig?,
 
+    @SerializedName("negativeNeutralButton")
```

Comment:
Not related to this PR but what does it mean 'negative neutral button'?

Reply:
It is a negative button but less intrusive

**Examples:**
Negative Button
![Negative Button](https://github.com/user-attachments/assets/f2a4dceb-769a-4e09-b5e0-41a73ad677be)

Negative Neutral Button
![Negative Neutral Button](https://github.com/user-attachments/assets/e1bc2790-06b8-4fee-8665-0c5f7d8a8af1)

Reply:
The name is confusing, indeed. Can we name it just neutral button or it's a commonly recognised name?

Reply:
We already have a neutral button, also most probably we will have a positive-neutral(no intrusive positive) button in future.

Reply:
Ah, okay then. Good to know.